lightning/src/ln/anchor_channel_configuration_tests.rs

```diff
@@ -0,0 +1,163 @@
+use crate::events::{Event, MessageSendEvent, MessageSendEventsProvider};
+use crate::ln::msgs::ChannelMessageHandler;
+use crate::ln::msgs::ErrorAction;
+use crate::ln::{functional_test_utils::*, ChannelId};
```

Comment:
nit: split this into two imports and alphabetize them. This should remove some unnecessary diff in the next commit too